common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
Did you try it without the call to `HTML()`?  If the message is built using the `HTML()` and `Text()` functions, it may be redundant.  I'm fine with whatever you decide, just pointing this out.